app/controllers/offline-controller.js

```diff
@@ -23,59 +22,25 @@ router.get('/x/offline-app-worker.js', (req, res, next) => {
         error.status = 404;
         next(error);
     } else {
-        res.set('Content-Type', 'text/javascript').send(getScriptContent());
+        // We add as few explicit resources as possible because the offline-app-worker can do this dynamically and that is preferred
+        // for easier maintenance of the offline launch feature.
+        const resources = config['themes supported']
+            .flatMap((theme) => [
+                `${config['base path']}${config['offline path']}/css/theme-${theme}.css`,
+                `${config['base path']}${config['offline path']}/css/theme-${theme}.print.css`,
+            ])
+            .concat([
+                `${config['base path']}${config['offline path']}/images/icon_180x180.png`,
+            ]);
+
+        const link = resources
+            .map((resource) => `<${resource}>; rel="prefetch"`)
+            .join(', ');
+
+        const script = fs.readFileSync(config.offlineWorkerPath, 'utf-8');
+
+        res.set('Content-Type', 'text/javascript');
+        res.set('Link', link);
+        res.send(script);
     }
 });
-
-/**
- * Assembles script contentå
- */
-function getScriptContent() {
-    // Determining hash every time, is done to make development less painful (refreshing service worker)
-    // The partialScriptHash is not actually required but useful to see which offline-app-worker-partial.js is used during troubleshooting.
```

Comment:
Prior to this change, it was possible to look at an open tab and tell what version of the frontend was being used. After this change, we won't be able to tell that but if we refresh we will know it's the latest. Is that right?

Will this limit our ability to know which frontend version we're working with when troubleshooting production issues?